migrations/v2/V202410301210__v2.12.UpdateListFields.sql

```diff
@@ -0,0 +1,16 @@
+UPDATE [${flyway:defaultSchema}].[Entityfield]
+SET    DefaultInView = 1
+WHERE  ID IN ( 'C94217F0-6F36-EF11-86D4-6045BDEE16E6', -- List.ID
```

Comment:
@JS-BC  why are we including ID, ListID, RecordID values as DefaultInView, I could see the case for the List's Name and in the ListDetail, the ListDetail.List - which is a vritual field back to List.Name, Sequence maybe as well, but why all the others. DefaultInView should be the thigns that end users find valuable and most of these are UUIDs which are very hard for an end user to make sense of.

Reply:
@AN-BC The ID field is included in many entities, I was under the assumption that was standard. For ListID and RecordID, youre right. It is better to use the virtual fields